src/lib/email.js

```diff
@@ -0,0 +1,27 @@
+import sendmail from 'sendmail';
+import StudentStore from '../store/student-store';
+
+const mailer = sendmail({
+    silent: true
+});
+
+export function sendMail(emails){
+    return new Promise((res, rej) => {
+        mailer({
+            from: 'testing@msoe.edu',
+            to: emails.join(','),
+            subject: 'ITEMS DUE SOON',
+            html: 'This is a test.'
+        }, err => {
+            if (err) return rej(err);
+            else res();
+        });
+    });
+}
+
+export function sendDueItemNotifications(){
+    let emails = StudentStore.getStudentsWithDueItems().map(student => student.email);
+    return this.sendMail(emails);
```

Comment:
Nit: Do we want to send one email to all of the students? Seems sub-optimal for privacy for students to be able to see the full list of other students with due items, but not too bad.